02.calendar/cal.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'date'
+
+opt = OptionParser.new
+opt.on('-m')
+opt.on('-y')
+
+opt.parse!(ARGV)
+month = ARGV[0] || Date.today.mon
+year = ARGV[1] || Date.today.year
+
+header = "#{month}月"+" "+year.to_s
+print header.center(20) + "\n"
+
+day_of_week = %w(日 月 火 水 木 金 土)
+  print " "+day_of_week.join(" ") + "\n"
+
+First_day = Date.new(year.to_i, month.to_i, 1)
+last_day = Date.new(year.to_i, month.to_i, -1)
+
+space = First_day.wday
+space.times do
+print "   "
+end
```

Comment:
```suggestion
print "   " * First_day.wday
```

[nits] `String#*` で文字列を繰り返すことができます。こういうところで使うのがおすすめです。